src/serverdlg.cpp

```diff
@@ -600,6 +600,12 @@ void CServerDlg::OnServerStopped()
 void CServerDlg::OnStopRecorder()
 {
     UpdateRecorderStatus ( QString::null );
+    if ( pServer->GetRecorderErrMsg() != QString::null )
```

Comment:
This is a bit ugly.  The recorder should have an `inError` state, exposed by the server as `GetRecorderInErrorState()` boolean.  I don't like checking for null strings as a state indicator...

Reply:
I'll take a look at that.

Reply:
I had a quick look. The `jamcontroller.*` part seemed manageable, but as the error messages are coming from `CJamRecorder::Init()`, this would have required refactoring as well (probably to use exceptions and catch them in the caller...?). As you say, there is already existing code working this way.

To avoid another round of reviews for that, I'd like to keep this out of this PR. Maybe we should have a new issue to track that?